tests/unit_tests/test_wave_02.py

```diff
@@ -34,7 +34,12 @@ def test_get_no_matching_items_by_category():
 
     items = vendor.get_by_category("electronics")
 
-    raise Exception("Complete this test according to comments below.")
+    assert len(items) == 0
+    assert item_a not in items
+    assert item_b not in items
+    assert item_c not in items
```

Comment:
This works, but we can make it simpler by removing lines 38-40. If the length is 0, it's impossible for any of the items to be in the list, so we don't need to check whether they're there.